### ISSUE TITLE
Default Username / Password not working

### ISSUE THREAD

**Issue by nicky4eyes** [com-py/Hibuntu#3]
Heya! I tried using the credentials you stated in your tutorial, but get denied every time.
**login: me
password: root**

Has this changed sometime or am I misinterpreting your instruction?

Thank you!

**Comment by com-py** [comment]
No, it has not changed. You can search the script and see that the $username is me and password is root.
Did the installation finish successfully? No error/warning?
